### ISSUE TITLE
investigate combining multiple metrics

### ISSUE THREAD

**Issue by leeoniya** [krausest/js-framework-benchmark#285]
@krausest 

what do you think about trying to measure multiple metrics per single pass? the most expensive part of all measurements is waiting on the dom to be created, so why do it a ton of times?

you could combine for example: startup -> 10k create -> 10k partial update -> 1k append -> 11k clear. same with 1k create -> 1k swap -> 1k delete

maybe even mem metrics can be folded into these.

i think this can reduce the overall benchmark time by 30-50%

**Comment by krausest** [comment]
I'll think about it. I really like that the benchmarks are independent right now, but maybe there's a clever way to keep that while adding your optimization.

**Comment by krausest** [comment]
I thought about it, but I reject it. I really like to keep them independent.
